src/proto/ctl/storage_nvme.proto

```diff
@@ -33,8 +33,8 @@ import "common.proto";
 message NvmeController {
 	// Namespace represents a namespace created on an NvmeController.
 	message Namespace {
-		int32 id = 1;		// namespace id
-		int32 size = 2;	// device capacity in GBytes
+		uint32 id = 1;		// namespace id
+		uint64 size = 2;	// device capacity in GBytes
```

Comment:
The comment isn't correct anymore, right? Should just be device capacity in bytes. Also looks like maybe there are missing updates to stuff under lib/spdk to accompany this change.